client/src/endpoint/account.rs

```diff
@@ -29,6 +29,48 @@ impl EndPoint for Details {
     }
 }
 
+/// An endpoint that returns a single value for a key/vaule pair associated with an account.
+///
+/// https://www.stellar.org/developers/horizon/reference/endpoints/data-for-account.html
+///
+/// ## Example
+/// ```
+/// use stellar_client::sync::Client;
+/// use stellar_client::endpoint::account;
+///
+/// let client      = Client::horizon_test().unwrap();
+/// let endpoint    = account::Data::new("GATLAI2D7SSH6PE3HXTDPTRM4RE5VRK6HGA63K5EWP75PSANCZRFDNB5", "Food");
+/// let record      = client.request(endpoint).unwrap();
+/// #
+/// # assert!(record.value() == "UGl6emE="); //
+/// ```
+#[derive(Debug)]
+pub struct Data {
+    id: String,
+    key: String,
+}
+
+impl Data {
+    /// Returns a new end point for account details. Hand this to the client in order to request
+    /// a single value for a key/value pair for an account.
+    pub fn new(id: &str, key: &str) -> Self {
+        Self {
+            id: id.to_string(),
+            key: key.to_string(),
```

Comment:
I believe these to_string() are redundant

Reply:
They won't be because you are passing in string references and need to convert to a `String` and take ownership.